src/server/scripts/Custom/Paymaster.cpp

```diff
@@ -0,0 +1,255 @@
+#include "AccountMgr.h"
+#include "ScriptMgr.h"
+#include "Player.h"
+#include "Config.h"
+#include "Common.h"
+#include "Chat.h"
+#include "ObjectAccessor.h"
+#include "ObjectMgr.h"
+#include "Player.h"
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "ScriptedGossip.h"
+#include "SharedDefines.h"
+#include "World.h"
+#include "WorldSession.h"
+#include "DataStores/DBCStores.h"
+#include "Spells/SpellMgr.h"
+#include <Commands/cs_learn.cpp>
```

Comment:
I'm not really a fan of including cpp files. In this particular case just copy `HandleLearnSkillRecipesHelper` to this file with its implementation and remove the cpp include.

Reply:
Resolved this and the include references in latest commit.  Tested in game, everything still works great